slackbridge/utils.py

```diff
@@ -124,7 +124,13 @@ def user_replace(match):
         """
         user_id = match.group(1)
         readable = match.group(2)
-        return readable or users[user_id].nickname
+
+        if readable or user_id in users:
+            return readable or users[user_id].nickname
+        elif user_id in bots:
+            return bots[user_id].nickname
+        else:
+            return 'unknown'
```

Comment:
I think you'd want this to return some kind of error, or at least make it clearer that it should never be hit, otherwise it could just show up in a message but wouldn't be too unusual, when we really shouldn't be ever hitting this case.

Reply:
I've added a comment. It's true that we never should be hitting it but assuming the regex happens to match it this 'catch-all' behavior is preferable to crashing in my opinion.